librz/core/cmd_windows_heap.c

```diff
@@ -35,4 +35,24 @@ RZ_IPI RzCmdStatus rz_cmd_debug_process_heaps_handler(RzCore *core, int argc, co
 RZ_IPI RzCmdStatus rz_cmd_debug_process_heap_block_handler(RzCore *core, int argc, const char **argv, RzOutputMode mode) {
 	NOT_SUPPORTED_ERROR_MESSAGE;
 }
+#endif
+
+/* API calls for Cutter */
+
```

Comment:
Since those aren't commands, what about moving them in a separate file? `librz/core/cheap.c` or something.

Reply:
sure I can do that